stream-chat-android-sample/src/main/kotlin/io/getstream/chat/sample/feature/channel/ChannelFragment.kt

```diff
@@ -79,8 +79,8 @@ class ChannelFragment : Fragment() {
                     is MessageListViewModel.Mode.Normal -> resetThread()
                 }
             }
-            binding.messageListView.setOnMessageEditHandler {
-                editMessage.postValue(it)
+            binding.messageListView.setOnMessageEditHandler { message ->
```

Comment:
You can use: binding.messageListView.setOnMessageEditHandler(::postMessageToEdit)